config/overlays/odh/inferenceservice-config-patch.yaml

```diff
@@ -60,7 +60,12 @@ data:
         "memoryRequest": "100Mi",
         "memoryLimit": "1Gi",
         "cpuRequest": "100m",
-        "cpuLimit": "1"
+        "cpuLimit": "1",
+        "headers": {
+          "propagate": [
+            "Authorization"
```

Comment:
@Jooho @danielezonca Please, advice if this is fine.
I don't see an issue as long as everything runs over TLS.

Reply:
I think it is a good idea to add token to downstream endpoints.